packages/blocks/src/api/registration.js

```diff
@@ -250,6 +250,27 @@ export function registerBlockType( name, settings ) {
 	return settings;
 }
 
+/**
+ * Registers a new block collection to group blocks in the same namespace in the inserter.
+ *
+ * @param {string} namespace      The namespace to group blocks by in the inserter; corresponds to the block namespace
+ * @param {Object} title, icon    The title to show in the inserter, The icon to show in the inserter
```

Comment:
Would be cool to try to type these like suggested by @aduth in https://github.com/WordPress/gutenberg/issues/18838